pandas/conftest.py

```diff
@@ -121,6 +124,13 @@ def pytest_runtest_setup(item):
         pytest.skip("skipping high memory test since --run-high-memory was not set")
 
 
+def pytest_collection_modifyitems(items):
+    for item in items:
+        # mark all tests in the pandas/tests/frame directory with "arraymanager"
+        if "/frame/" in item.nodeid:
```

Comment:
ill this be posix-specific?

Reply:
Looking at the output of the Azure windows build, it seems pytest uses "consistent" formatting with forward slashes in the name of the test. 
(and the extra tests were running there, so it's working)